src/pages/HomePage.tsx

```diff
@@ -156,6 +161,20 @@ class HomePage extends React.Component<Props, State> {
         console.log(this.state.data);
       }).catch((e: any) => { console.log(e) })
     }
+
+    else if (this.props.isNotes === "true") {
+      console.log(this.props.match.params.note)      
+      const getNotes:any = API.graphql({
+        query: queries.getNotes,
+        variables: { id: this.props.match.params.note },
+        authMode: GRAPHQL_AUTH_MODE.API_KEY
+      });
+      getNotes.then((json: any) => {
+        console.log({ "Success queries.getNotes: ": json });
+        this.setState({ data: json.data.getNotes })
+        console.log(this.state.data);
+      }).catch((e: any) => { console.log(e) })
```

Comment:
Use `console.error` for logging errors. You can also probably use the `Error` type for `e`